packages/overlays/src/OverlayController.js

```diff
@@ -358,11 +357,14 @@ export class OverlayController {
   __setupTeardownAccessibility({ phase }) {
     if (phase === 'init') {
       this.__storeOriginalAttrs(this.contentNode, ['role', 'id']);
```

Comment:
Should we guard this.contentNode as well?

Reply:
It already is: https://github.com/ing-bank/lion/blob/master/packages/overlays/src/OverlayController.js#L249